### PR TITLE
Biserver 12069

### PR DESCRIPTION
**Pull request by ivanpogodin**

### CODE REVIEW

**Comment by ivanpogodin** [comment]
@pamval , @tmorgner , @rmansoor , could you review it?

**Comment by pamval** [comment]
@rfellows maybe you can review this one?

**Comment by rfellows** [comment]
I'd like to see a unit test or two. but the code itself looks fine to me.

**Comment by dkincade** [comment]
@rfellows - great comment about wanting unit tests - keep up the good reviews and the push for completeness!

File: extensions/src/org/pentaho/platform/plugin/action/mdx/MDXBaseComponent.java [hunk 1]
```diff
@@ -593,4 +597,20 @@ protected void setConnection( IPentahoConnection connection ) {
...
+  boolean isCatalogVfsAccepted( String catalog ) {
+    try {
+      FileObject f = org.apache.commons.vfs2.VFS.getManager().resolveFile( catalog );
+      return ( f != null );
+    } catch ( Exception e ) {
+      Log logger = getLogger();
+      if ( logger != null ) {
+        logger.trace( "Catalog is not accessible: " + catalog, e );
+      } else {
+        System.out.println( "TRACE: Catalog is not accessible: " + catalog );
```
Comment (lucboudreau) [inline extensions/src/org/pentaho/platform/plugin/action/mdx/MDXBaseComponent.java]: Please don't output to System.out.

**Comment by ivanpogodin** [comment]
@lucboudreau , @rfellows , 
added tests
got rid of System.out

**Pull request merged**